corpus/accounts/urls.py

```diff
@@ -12,7 +12,8 @@
     path("signup/", signup, name="accounts_signup"),
     path("login/", signin, name="accounts_signin"),
     path("logout/", signout, name="accounts_signout"),
-    path("reset/", PasswordResetView.as_view(), name="password_reset"),
+    path("reset/", PasswordResetView.as_view(html_email_template_name="../templates/registration/password_reset_email.html"),
```

Comment:
You don't need to give the relative path - Django will automatically search in the templates directory. Just use `registration/password_reset_email.html`

Reply:
It is only searching in the accounts/reset directory for some reason? When I just use `registration/password_reset_email.html` it shows TemplateDoesNotExist at /accounts/reset/

Reply:
This seems to work on my end, please make the change. We'll test on prod, and if it breaks, we'll fallback to this.